examples/test/post-processing/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Post-processing</title>
+    <meta name="description" content="Post-processing — A-Frame">
+    <script src="../../../dist/aframe.js"></script>
+    <script src="components/pulse.js"></script>
+    <script src="components/envmap.js"></script>
+    <script src="components/bloom.js"></script>
+    <script src="components/compositor.js"></script>   
+    <script src="components/bloom-toggler.js"></script>
+  </head>
+  <body>
+    <a-scene post-process="" bloom-toggler="">
```

Comment:
If we reuse the register shader API to register effects one could do 
````html
<a-scene post-process="effects: bloom, nightVision; bloomIntensity: 1.0; nightVisionIntensity: 0.6">...</a-scene>
````
The name of the effect is prepended to the attribute names defined in the schema of the effect (aka shader)